pkg/util/kube/pvc_pv.go

```diff
@@ -74,6 +74,10 @@ func DeletePVAndPVCIfAny(ctx context.Context, client corev1client.CoreV1Interfac
 	if err := EnsureDeletePVC(ctx, client, pvcName, pvcNamespace, ensureTimeout); err != nil {
 		log.Warnf("failed to delete pvc %s/%s with err %v", pvcNamespace, pvcName, err)
 	}
+
+	if err := EnsureDeletePV(ctx, client, pvcObj.Spec.VolumeName, ensureTimeout); err != nil {
+		log.Warnf("pv %s was not removed with err %v", &pvcObj.Spec.VolumeName, err)
```

Comment:
Why do we pass a pointer (of `pvcObj.Spec.VolumeName`) here for `%s`?

Reply:
good catch..
```suggestion
		log.Warnf("pv %s was not removed with err %v", pvcObj.Spec.VolumeName, err)
```